tests/e2e/map/map_spatial.spec.js

```diff
@@ -31,764 +52,1767 @@ test.describe.skip('Map: Spatial interactions', () => {
+
+          // Wait for map animation to complete
+          await page.waitForTimeout(250)
+
+          // Draws the spatial on the map
+          await expect(page).toHaveScreenshot('4326-box-spatial-typed.png', {
+            clip: screenshotClip
+          })
+        })
       })
-    })
 
-    test.describe('When editing a point', () => {
-      test('renders correctly', async ({ page }) => {
-        await interceptUnauthenticatedCollections({
-          page,
-          body: commonBody,
-          headers: commonHeaders,
-          additionalRequests: [{
-            body: pointBody,
-            headers: {
-              ...commonHeaders,
-              'cmr-hits': '2'
-            },
-            paramCheck: (parsedQuery) => parsedQuery?.point?.[0]?.match(/42\.\d+,-7\.\d+/)
-          },
-          {
-            body: pointBodyEdited,
-            headers: {
-              ...commonHeaders,
-              'cmr-hits': '3'
+      test.describe.skip('When editing a bounding box', () => {
+        test('renders correctly', async ({ page }) => {
+          await interceptUnauthenticatedCollections({
+            page,
+            body: commonBody,
+            headers: commonHeaders,
+            additionalRequests: [{
+              body: boundingBoxBody,
+              headers: {
+                ...commonHeaders,
+                'cmr-hits': '2'
+              },
+              paramCheck: (parsedQuery) => parsedQuery?.bounding_box?.[0]
+                ?.match(/42\.\d+,-21\.\d+,56\.\d+,-7\.\d+/)
             },
-            paramCheck: (parsedQuery) => parsedQuery?.point?.[0]?.match(/42\.\d+,-24\.\d+/)
-          }]
+            {
+              body: boundingBoxBodyEdited,
+              headers: {
+                ...commonHeaders,
+                'cmr-hits': '3'
+              },
+              paramCheck: (parsedQuery) => parsedQuery?.bounding_box?.[0]
+                ?.match(/42\.\d+,-21\.\d+,56\.\d+,6\.\d+/)
+            }]
+          })
+
+          await page.goto('/')
+
+          // Select the bounding box spatial type
+          await page.getByRole('button', { name: 'spatial-selection-dropdown' }).click()
+          await page.getByRole('button', { name: 'Rectangle' }).click()
+
+          // Add the bounding box to the map
+          await page.mouse.click(1000, 500)
+          await page.mouse.click(1100, 600)
+
+          // Updates the URL
+          await expect(page).toHaveURL(/search\?sb\[0\]=42\.\d+%2C-21\.\d+%2C56\.\d+%2C-7\.\d+/)
+
+          // Populates the spatial display field
+          await expect(page.getByTestId('spatial-display_southwest-point')).toHaveValue(/-21\.\d+,42\.\d+/)
+          await expect(page.getByTestId('spatial-display_northeast-point')).toHaveValue(/-7\.\d+,56\.\d+/)
+
+          // Checking that the right number of results are loaded ensures that the route
+          // was fulfilled correctly with the succesfull paramCheck
```

Comment:
nit;
```suggestion
          // was fulfilled correctly with the successful paramCheck
```
do global replace